Xamarin.PropertyEditing.Mac/Controls/Custom/PropertyButton.cs

```diff
@@ -154,8 +164,8 @@ private void PopUpContextMenu ()
 
 		private void FocusClickedRow ()
 		{
-			if (Superview != null) {
-				MakeFocusableKeyViewFirstResponder (Superview.Subviews);
+			if (Superview is EditorContainer ec) {
```

Comment:
By only passing the `EditorView.NativeView.Subviews` the recursion only traverses down the `EditorContainer` tree, thus avoiding the `Variation` buttons.